src/pages/Authentication/Login/Login.jsx

```diff
@@ -0,0 +1,15 @@
+import { Footer, Header } from "../../../components";
+import { useDocumentTitle } from "../../../customHooks/useDocumentTitle";
+import "./Login.css";
+
+export const Login = () => {
+	useDocumentTitle("Login | MS");
+
+	return (
+		<div className="body--homepage">
+			<Header />
+			<h2>Login</h2>
+			<Footer />
```

Comment:
you can use header and footer outside your routes to prevent the unnecessary rerenders of header and footer.

Reply:
Good suggestion. Thanks for Review.